docs/src/pages.js

```diff
@@ -26,6 +26,7 @@ datagrid.children =
           title: 'Overview',
         },
         { pathname: '/components/data-grid/getting-started' },
+        { pathname: '/components/data-grid/demo' },
```

Comment:
I can't remember what we said about this. Should "Demo" be before "Getting Started" or after?

Reply:
I think we can move it above the "Getting Started" section, makes more sense to check the features before deciding to implement the grid.